README.md

```diff
@@ -26,9 +26,26 @@ To build .NET Core 3.1 from source, pick a specific Git tag from this repo with
 
 ## Source-build goals
 
-The key goal of source-build is to satisfy the official packaging rules of commonly used Linux distributions, such as [Fedora](https://fedoraproject.org/wiki/Packaging:Guidelines) and [Debian](https://www.debian.org/doc/manuals/maint-guide/build.en.html). Many Linux distributions have similar rules. These rules tend to have two main principles: consistent reproducibility, and source code for everything.
+There are two primary goals of the source-build effort:
+
+1. Increase .NET adaption by focusing on making .NET available everywhere
+
+   If .NET was available everywhere - including Linux distributions and package managers like Homebrew - as a first class citizen it, would make .NET a more attractive option for developers who might otherwise look at other languages or runtimes. Users would be more likely to start using and keep using .NET if .NET is available on their development and release platforms of choice.
```

Comment:
```suggestion
   If .NET was available everywhere - including Linux distributions and package managers like Homebrew - as a first class citizen, it would make .NET a more attractive option for developers who might otherwise look at other languages or runtimes. Users would be more likely to start using and keep using .NET if .NET is available on their development and release platforms of choice.
```